### PR TITLE
Language param on EntityLoadMultiple data producer shouldn't be multiple

### PR DESCRIPTION
**Pull request by juanjol**

### CODE REVIEW

File: src/Plugin/GraphQL/DataProducer/Entity/EntityLoadMultiple.php [hunk 1]
```diff
@@ -31,7 +31,6 @@
  *     ),
  *     "language" = @ContextDefinition("string",
  *       label = @Translation("Entity languages"),
```
Comment (klausi) [inline src/Plugin/GraphQL/DataProducer/Entity/EntityLoadMultiple.php]: We should also fix the label here to "Entity language"

**Review by klausi — CHANGES_REQUESTED** [review]
Thanks, just one minor thing.

File: src/Plugin/GraphQL/DataProducer/Entity/EntityLoadMultiple.php (hunk 1 above)
Comment (juanjol) [reply src/Plugin/GraphQL/DataProducer/Entity/EntityLoadMultiple.php]: Label updated

**Comment by klausi** [comment]
Thank you!

**Pull request merged**